extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/thing/LightlevelSensor.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hue" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Hue Light Level Sensor-->
+	<thing-type id="0106">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>Hue Lightlevel Sensor</label>
+		<description>A sensor providing lightlevel information</description>
+
+		<channels>
+			<channel id="lightlevel" typeId="lightlevel" />
+			<channel id="dark" typeId="dark" />
+			<channel id="daylight" typeId="daylight" />
+		</channels>
+
+		<representation-property>uniqueId</representation-property>
+
+		<config-description>
+			<parameter name="lightId" type="text">
+				<label>Sensor ID</label>
+				<description>The identifier that is used within the hue bridge.</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="tholddark" type="integer">
```

Comment:
Is the user supposed to change these values? If so it does not work. It is not send to the Hue Bridge.